src/vs/vscode.d.ts

```diff
@@ -1616,6 +1616,11 @@ declare module 'vscode' {
 		 */
 		matchOnDetail?: boolean;
 
+		/**
+		 * An optional flag to sort the final results by index of first query match in label, defaults to true.
+		 */
+		sortByLabel?: boolean;
```

Comment:
These changes should be in vscode.proposed.d.ts, new API starts there until we decide to release it as stable API. More info: https://github.com/Microsoft/vscode/wiki/Extension-API-process